### PR TITLE
feat(subgraph,web): fix coherent votes at vote level instead of dispute level

### PR DESCRIPTION
**Pull request by kemuru**
works good.

image

image

## Release Notes

- **New Features**
  - Enhanced user tracking with new metrics: `totalCoherentVotes`, `totalResolvedVotes`, and `coherenceScore`.
  - Introduced a new voting mechanism, `ClassicVote`, for improved dispute resolution.

- **Improvements**
  - Streamlined user dispute resolution process by simplifying parameters and removing unnecessary complexity.
  - Updated various components to reflect new naming conventions for user voting metrics, enhancing clarity.

- **Configuration**
  - Updated version of the Kleros v2 core subgraph from `0.7.4` to `0.7.6`.

---

## PR-Codex overview
This PR primarily focuses on renaming properties related to user voting metrics from `totalCoherent` to `totalCoherentVotes` and updating the associated logic throughout the application. It also enhances the user level calculation by introducing minimum disputes criteria.

### Detailed summary
- Renamed `totalCoherent` to `totalCoherentVotes` in various files.
- Added `totalResolvedVotes` in multiple components.
- Updated `getUserLevelData` to include `totalResolvedDisputes`.
- Modified `computeCoherenceScore` to use `totalCoherentVotes` and `totalResolvedVotes`.
- Adjusted various components to reflect the new properties and ensure proper rendering.
- Updated `resolveUserDispute` function to simplify parameters and logic.
- Enhanced user level criteria with minimum disputes requirements.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

File: web/src/pages/Home/TopJurors/JurorCard/index.tsx [hunk 1]
```diff
@@ -7,12 +7,12 @@ interface IJurorCard {
   rank: number;
   address: `0x${string}`;
   coherenceScore: number;
-  totalCoherent: number;
-  totalResolvedDisputes: number;
+  totalCoherentVotes: number;
+  totalResolvedVotes: number;
```
[the remark this answers is not in the document]
Comment (kemuru) [reply web/src/pages/Home/TopJurors/JurorCard/index.tsx:11]: not true. totalResolvedDisputes still persists

File: subgraph/core/src/KlerosCore.ts [hunk 2]
```diff
@@ -127,6 +127,42 @@ export function handleNewPeriod(event: NewPeriod): void {
...
+        if (vote.choice === null) continue;
+
+        // Check if the vote choice matches the final ruling
+        if (vote.choice!.equals(dispute.currentRuling)) {
+          juror.totalCoherentVotes = juror.totalCoherentVotes.plus(ONE);
+        }
+
+        // Recalculate coherenceScore
+        if (juror.totalResolvedVotes.gt(ZERO)) {
+          const coherenceScore = juror.totalCoherentVotes.times(BigInt.fromI32(100)).div(juror.totalResolvedVotes);
```
Comment (alcercu) [inline subgraph/core/src/KlerosCore.ts:159]: it's missing the smoothing factor, check [here](https://github.com/kleros/kleros-v2/pull/1702/files#diff-e01d09c5f53e6045158d43b591c1868293037842fa12534f31d5433f4bf8bdcaL8)

**Review by alcercu — CHANGES_REQUESTED** [review]

**Review by alcercu — DISMISSED** [review]
lgtm

**Review by alcercu — APPROVED** [review]
lgtm

**Pull request merged**